packages/template-retail-react-app/app/pages/product-list/index.test.js

```diff
@@ -139,6 +139,64 @@ test('clicking a filter will change url', async () => {
     )
 })
 
+test('clicking a filter on mobile or desktop applies changes to both', async () => {
+    window.history.pushState({}, 'ProductList', '/uk/en-GB/category/mens-clothing-jackets')
+    const {user} = renderWithProviders(<MockedComponent />, {
+        wrapperProps: {siteAlias: 'uk', locale: {id: 'en-GB'}}
+    })
+    // NOTE: Look for a better wait to wait an additional render.
+    await waitFor(() => !!screen.getByText(/Beige/i))
+
+    // Only desktop filters should be present
+    // Test using two buttons since there was a bug where using only one filter would properly
+    // apply changes to both desktop and mobile, but 2 or more would cause it to fail
+    let beigeBtns = screen.getAllByLabelText('Beige (6)')
+    let blueBtns = screen.getAllByLabelText('Blue (27)')
```

Comment:
```suggestion
    // Only desktop filters should be present
    // Test using two buttons since there was a bug where using only one filter would properly
    // apply changes to both desktop and mobile, but 2 or more would cause it to fail
    let beigeBtns = await screen.findAllByLabelText('Beige (6)')
    let blueBtns = await screen.findAllByLabelText('Blue (27)')
```
Per the [docs](https://testing-library.com/docs/queries/about#types-of-queries), `findBy` is just `getBy` wrapped in `waitFor`, so I thiiink this should be doing what you want?

Reply:
Unfortunately this does not work :'(